### ISSUE TITLE
nix hash out of date

### ISSUE THREAD

**Issue by skbolton** [lexical-lsp/lexical#614]
Looks like #505 is an issue again. I will make an attempt to fix but wanted to call it out.

**Comment by scohen** [comment]
thanks @skbolton 
@hauleth please help, is there something I need to do when releasing?

**Comment by hauleth** [comment]
I will update it ASAP. Maybe we should move the Nix flake to separate repo to make it a little bit easier…

**Comment by hauleth** [comment]
However I tested with current master and it works for me… @skbolton how do you build?

**Comment by skbolton** [comment]
Its a flake.nix in on of our projects that I was doing a `nix flake update` on when I saw the 0.5 release. Let me try again and make sure I didn't do something wrong.

**Comment by skbolton** [comment]
One of my machines just worked so not sure why the other failed. I will look into that and can create another new issue. Also we can close this unless my previous comment is something to worry about.

**Comment by hauleth** [comment]
@scohen I am preparing script to make it even easier for non-Nix users to update hash. I want also to test if I can use some Docker image to allow updating hash even without Nix installed locally.
